### ISSUE TITLE
How to hide left side menu?

### ISSUE THREAD

**Issue by RapFan89** [DanielnetoDotCom/YouPHPTube#1446]
Hi there,
i want to hide the whole left side menu (the sandwich button next to the logo image). Is that possible? Thank you.

**Comment by Andiweb87** [comment]
If im not wrong there is not an option to hide the left side menu but in plugins --> CustomizeAdvanced (CustomizeAdvanced)  you can hide some content in the left side menu but not all menu.

**Comment by RapFan89** [comment]
Yes but i need to hide the categories, and there's no option to do that there.

**Comment by Andiweb87** [comment]
Is an option to do that, go again to plugins --> CustomizeAdvanced (CustomizeAdvanced) and find the option show in screenshot

https://snag.gy/Tb9o7a.jpg

**Comment by RapFan89** [comment]
i do not have it 
![immagine](https://user-images.githubusercontent.com/20926927/52222708-712c7f80-28a4-11e9-86f6-2fceb23e9bf9.png)

**Comment by RapFan89** [comment]
oh sorry i had to sync master, now it's ok

**Comment by Andiweb87** [comment]
great , you can close the issue now if you dont have any more question.
